docs/dxp/latest/en/developing-applications/core-frameworks/configurable-application/ddm-form-annotations/resources/liferay-v1d9.zip/v1d9-impl/src/main/java/com/acme/v1d9/internal/configuration/admin/definition/V1D9ConfigurationDDMFormDeclaration.java

```diff
@@ -0,0 +1,20 @@
+package com.acme.v1d9.internal.configuration.admin.definition;
+
+import com.liferay.configuration.admin.definition.ConfigurationDDMFormDeclaration;
+
+import org.osgi.service.component.annotations.Component;
+
+@Component(
+	immediate = true,
```

Comment:
@mtokudome Can you test removing `immediate = true` and let me know if it still works as expected?

Reply:
Good catch, I removed the line and this still works as expected